components/v1/MarketDetailPage/ButtonDepositOrWithdraw.tsx

```diff
@@ -27,9 +27,9 @@ const ButtonDepositOrWithdraw: FunctionComponent<ButtonDepositOrWithdrawProps> =
         <Dialog.Root>
             <Dialog.Trigger className="w-full w-full rounded-full border border-blue-light-11 bg-blue-light-10 py-[11px] text-sm font-semibold leading-4 tracking-[-0.02em] text-gray-light-1 dark:border-blue-dark-11 dark:bg-blue-dark-10 dark:text-blue-light-1">Deposit or Withdraw</Dialog.Trigger>
             <Dialog.Overlay className="fixed inset-0 z-30 bg-gray-dark-1/60 backdrop-blur dark:bg-black/60" />
-            <Dialog.Content className="fixed left-0 bottom-0 z-30 w-screen sm:min-h-[477px] sm:-translate-y-1/3">
+            <Dialog.Content className="fixed left-0 bottom-0 z-30 flex h-screen w-screen items-center">
```

Comment:
maybe we can use `sm:items-center` instead?

Reply:
Okay thank you for the review sir! I think it is fixed now..